test/run-stp-tests.sh

```diff
@@ -268,7 +268,7 @@ case "$usedebugprintfs" in
 		echo "Enabling debug printfs (level $usedebugprintfs); log files can be found in $BASE/log..."
 		CUPS_DEBUG_LOG="$BASE/log/debug_printfs.%d"; export CUPS_DEBUG_LOG
 		CUPS_DEBUG_LEVEL="$usedebugprintfs"; export CUPS_DEBUG_LEVEL
-		CUPS_DEBUG_FILTER='^(http|_http|ipp|_ipp|cups.*Request|cupsGetResponse|cupsSend|mime).*$'; export CUPS_DEBUG_FILTER
+		CUPS_DEBUG_FILTER='^(http|_http|ipp|_ipp|cups.*Request|cupsGetResponse|cupsSend).*$'; export CUPS_DEBUG_FILTER
```

Comment:
Not sure I understand the purpose of this change? The MIME functions are a common source of problems which is why they are included in the filter regex.

Reply:
This is 5d3c5a4928acb97dc069883e48aed78bfe80e61a, which is a revert of 830cfed95a5349b008999eebd34f5233bc35829c.

In other words: `|mime` was added to test `pdftourf`, which Linuxes don't have.

Reply:
This is 5d3c5a4928acb97dc069883e48aed78bfe80e61a, which is a revert of 830cfed95a5349b008999eebd34f5233bc35829c.

In other words: `|mime` was added to test `pdftourf`, but Linuxes don't have it.